src/main/kotlin/no/skatteetaten/aurora/boober/service/RedeployService.kt

```diff
@@ -54,8 +53,7 @@ class RedeployService(val openShiftClient: OpenShiftClient,
             return createFailedRedeployResult(imageStreamTagResponse.exception, openShiftResponses)
         }
 
-        imageStreamTagFromJson(imageStreamTagResponse.responseBody).findErrorMessage()
-                ?.let { return createFailedRedeployResult(it, openShiftResponses) }
+        imageStreamTagResponse.responseBody?.at("/image/dockerImageMetadata") ?: return createFailedRedeployResult("Redeploy failed", openShiftResponses)
```

Comment:
Ser ut som det mangler noe i Kubernetes DSL, den ville mappe dockerImageMetadata til ImageStreamTag objektet. Vet ikke om dette er god nok sjekk for å vite om det var en suksessfull deploy.

Reply:
Skjønte ikke helt problemet her, hvorfor skapte dette et problem?

Når det gjelder requesten som sendes, har vi fått med alt som trengs (i ImageStreamTagGenerator)?